h2o-core/src/main/java/hex/schemas/HyperSpaceSearchCriteriaV99.java

```diff
@@ -3,6 +3,8 @@
 import hex.ScoreKeeper;
 import hex.grid.HyperSpaceSearchCriteria;
 import water.api.API;
+import water.api.EnumValuesProvider;
+import water.api.schemas3.ModelParamsValuesProviders.StoppingMetricValuesProvider;
```

Comment:
using the values provider in this class as I was creating some of them for `AutoMLStoppingCriteriaV99`